### PR TITLE
add redirect_immediate param

### PR DESCRIPTION
**Pull request by safriks**
## Related Issue
 no redirect_immediate param in this library

## Proposed Changes
 add the param

### CODE REVIEW

File: lib/nordigen_ruby/api/requisitions.rb [hunk 1]
```diff
@@ -9,14 +9,15 @@ def initialize(client)
...
-        def create_requisition(redirect_url:, reference:, institution_id:, user_language: "en", agreement: nil, account_selection: false)
+        def create_requisition(redirect_url:, reference:, institution_id:, user_language: "en", agreement: nil, account_selection: false, redirect_immediate: false)
             # Create requisition. For creating links and retrieving accounts.
             payload = {
                 "redirect": redirect_url,
                 "reference": reference,
                 "institution_id": institution_id,
                 "user_language": user_language,
                 "account_selection": account_selection,
+                "redirect_immediate" redirect_immediate
```
Comment (verlinden) [inline lib/nordigen_ruby/api/requisitions.rb:20]: @victory-sokolov  this introduced a bug in the library who is now crashing. `:` is missing in `"redirect_immediate": redirect_immediate`

Comment (victory-sokolov) [reply lib/nordigen_ruby/api/requisitions.rb:20]: @verlinden Hey! Thanks for noticing this, my bad i didn't noticed this, creating fix [#21 ](https://github.com/nordigen/nordigen-ruby/pull/21)

Comment (victory-sokolov) [reply lib/nordigen_ruby/api/requisitions.rb:20]: Released fix, please take a look if it is working now.
Thank you!

Comment (verlinden) [reply lib/nordigen_ruby/api/requisitions.rb:20]: @victory-sokolov thanks, it's working 👍

**Pull request merged**